packages/@react-aria/datepicker/src/useDateSegment.ts

```diff
@@ -23,7 +22,7 @@ import {useSpinButton} from '@react-aria/spinbutton';
 
 export interface DateSegmentAria {
   /** Props for the segment element. */
-  segmentProps: DOMAttributes
+  segmentProps: React.HTMLAttributes<HTMLDivElement>
```

Comment:
why change this?

Reply:
There was a warning with the original type.

https://github.com/adobe/react-spectrum/blob/2964e9aafc281888cd65da5bd77512e7f999bd84/packages/%40react-aria/datepicker/src/useDateSegment.ts#L368-L397
<img width="852" alt="image" src="https://github.com/adobe/react-spectrum/assets/76232929/67be7ede-0c89-46af-bab4-3114fc75aedf">

Reply:
huh odd it's not coming up for me, I wonder if when we turn on strict mode that this will show up

Reply:
Hmmm, maybe it is.